tests/unit/src/components/InputsOutputs.spec.js

```diff
@@ -0,0 +1,50 @@
+import { shallowMount } from '@vue/test-utils'
```

Comment:
This is imported globally in setup.js
```suggestion
import { shallowMount } from '@vue/test-utils'
```